tests/Block/ContactTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Wearesho\Bobra\Ubki\Tests\Block;
+
+use Carbon\Carbon;
+
+use Wearesho\Bobra\Ubki;
+
+/**
+ * Class ContactTest
+ * @internal
+ * @package Wearesho\Bobra\Ubki\Tests\Block
+ */
+class ContactTest extends Ubki\Tests\Extend\BlockTestCase
+{
+    /** @var Ubki\Block\Contact */
+    protected $block;
+
+    protected function setUp(): void
+    {
+        Carbon::setTestNow(Carbon::now());
```

Comment:
setTestNow should not be used in this case. Just use const with static timestamp.
Also, if u use setTestNow(time) you have to do `setTestNow()` in tearDown